### PR TITLE
Feature: Support for custom runtime configurations

### LINKED ISSUE #114: Clarify where velocitas.json has to be
Currently we require that the velocitas.json is located in the root of the repository, but we miss some clear definition what happens if velocitas init is called in another folder e.g.

### PR DESCRIPTION
**Pull request by doosuu**
This PR adds the ability to overwrite `runtime.json` locally from within a project directory. This can be used to completely re-define the runtime or add further services/providers.

### CODE REVIEW

File: velocitas_lib/velocitas_lib.py [hunk 1]
```diff
@@ -67,9 +76,21 @@ def get_cache_data() -> dict[str, Any]:
...
+    variable_value = require_env("runtimeFilePath")
+
+    if variable_value is not None:
+        overwritten_path = Path(variable_value)
+        if not overwritten_path.is_absolute():
+            overwritten_path = Path(get_workspace_dir()).joinpath(overwritten_path)
+
+        if overwritten_path.exists():
+            path = overwritten_path
+            print(f"Runtime.json path redirected to {path}")
```
Comment (MP91) [inline velocitas_lib/velocitas_lib.py]:
```suggestion
            print(f"runtime.json path redirected to {path}")
```

File: velocitas_lib/velocitas_lib.py [hunk 2]
```diff
@@ -67,9 +76,21 @@ def get_cache_data() -> dict[str, Any]:
 
 def get_services() -> dict[str, Any]:
     """Return all specified services as Python object."""
+    path = f"{get_package_path()}/runtime.json"
+    variable_value = require_env("runtimeFilePath")
+
+    if variable_value is not None:
+        overwritten_path = Path(variable_value)
+        if not overwritten_path.is_absolute():
+            overwritten_path = Path(get_workspace_dir()).joinpath(overwritten_path)
```
Comment (MP91) [inline velocitas_lib/velocitas_lib.py:85]: this only works under the assumption that the velocitas.json is in the root of the repo or is the workspace_dir always the directory containing the velocitas.json? do we have that requirement somewhere?

**Review by MP91 — CHANGES_REQUESTED** [review]

File: velocitas_lib/velocitas_lib.py (hunk 2 above)
Comment (doosuu) [reply velocitas_lib/velocitas_lib.py:85]: By CLI design, the `.velocitas.json` always lives in the WORKSPACE_ROOT. It identifies the project as a Velocitas project.

Comment (MP91) [reply velocitas_lib/velocitas_lib.py:85]: Issue in CLI added to make this more clear: https://github.com/eclipse-velocitas/cli/issues/114

**Review by MP91 — APPROVED** [review]
LGTM

**Pull request merged**